pkg/hooks/read.go

```diff
@@ -95,4 +96,6 @@ func ReadDir(path string, extensionStages []string, hooks map[string]*current.Ho
 
 func init() {
 	Readers[current.Version] = current.Read
+	Readers[old.Version] = old.Read
+	Readers[""] = old.Read
```

Comment:
Well, it works, I guess, but this usage is new to me so of course I had to edit `/usr/share/containers/oci/hooks.d/oci-umount.json` and add `"version": "xx",` for several values of `xx`. Results:
| Version | Error |
| :--- | :--- |
| "0.1.0" | works fine |
| "0.1.1" | error setting up OCI Hooks ... unrecognized hook version: "0.1.1" |
| "1.0.0" | error setting up OCI Hooks ... 1.0.0: json: cannot unmarshal string into Go struct field Hook.hook of type specs.Hook |

I'll buy that my `1.0.0` is my own problem, probably because the format has changed somehow and I can't just add "version" without changing other things.

But, is there ever a possibility of a version other than `0.1.0`, `1.0.0`, or (absent)? Is there a way to specify a fallback `Reader`?

Reply:
Sure, it's code.  I will default to current.Read

Reply:
Not sure if this helps...but I just tried editing `/usr/share/containers/oci/hooks.d/oci-umount.json` and add `"version": "0.1.0",` with result:

`Error: error setting up OCI Hooks: parsing hook "/usr/share/containers/oci/hooks.d/oci-umount.json": unrecognized hook version: "0.1.0"`

Reply:
@heatmiser Try that with my patch and it should work.